frontend/src/Components/ConvertSeconds.tsx

```diff
@@ -0,0 +1,17 @@
+const convertSeconds = (secs: number) => {
```

Comment:
Not even a nit: just a thought

Personally, I am partial to terse code so I would do something like
```ts
const convertSeconds = (secs: number) => {
  const hours = Math.floor(secs / 3600);
  const minutes = Math.floor((secs % 3600) / 60);
  const seconds = Math.floor(secs % 60);

  return hours ? { number: hours, label: `hour${hours === 1 ? '' : 's'}` }
       : minutes ? { number: minutes, label: `minute${minutes === 1 ? '' : 's'}` }
       : { number: seconds, label: `second${seconds === 1 ? '' : 's'}` };
}; export default convertSeconds;
```
the only reason I say is because I see that some braces are absent to make it more concise, and I have found leaving them out (usually in c/c++) can be a footgun because it might not be obvious to the next person. That's entirely up to you either way tho. I just gotta comment on something I guess :)

Reply:
No definitely way better do do it this way... I'll get that changed. Thanks!